src/rpc/common/Validators.h

```diff
@@ -340,4 +344,23 @@ class CustomValidator final
     verify(boost::json::value const& value, std::string_view key) const;
 };
 
+/**
+ * @brief Helper function to check if sv is an uint32 number or not
+ */
+[[nodiscard]] bool
+checkIsU32Numeric(std::string_view sv);
+
+/**
+ * @brief Provide a common used validator for ledger hash
+ * LedgerHash must be a string and hex
+ */
+extern CustomValidator LedgerIndexValidator;
+
+/**
+ * @brief Provide a common used validator for ledger index
+ * LedgerIndex must be a string or int
+ * If it is string, it must be "validated" or "int"
```

Comment:
The 'or "int"' part makes me think it's a literal string `"int"`. 
We should say something like:
```
If the specified LedgerIndex is a string, it's value must be either "validated" or a valid integer value represented as a string.
```